public/app/plugins/datasource/grafana-azure-monitor-datasource/azure_log_analytics/azure_log_analytics_datasource.ts

```diff
@@ -134,6 +134,7 @@ export default class AzureLogAnalyticsDatasource extends DataSourceWithBackend<
     const query = templateSrv.replace(item.query, scopedVars, interpolateVariable);
 
     return {
+      ...target,
       refId: target.refId,
```

Comment:
you can know remove this line (the same for others)

Reply:
I may delete this PR in favour of #49543

Reply:
I still think that having both contributes to more robust code :)